.github/workflows/integ-test-inject-develocity.yml

```diff
@@ -12,21 +12,21 @@ on:
         type: boolean
         default: false
     secrets:
-      GRADLE_ENTERPRISE_ACCESS_KEY:
+      DEVELOCITY_ACCESS_KEY:
         required: true
 
 env:
   DOWNLOAD_DIST: ${{ inputs.download-dist }}
   GRADLE_BUILD_ACTION_CACHE_KEY_PREFIX: provision-gradle-versions-${{ inputs.cache-key-prefix }}
   GRADLE_BUILD_ACTION_CACHE_DEBUG_ENABLED: true
-  GRADLE_ENTERPRISE_INJECTION_ENABLED: true
-  GRADLE_ENTERPRISE_URL: https://ge.solutions-team.gradle.com
-  GRADLE_ENTERPRISE_PLUGIN_VERSION: 3.16.1
-  GRADLE_ENTERPRISE_CCUD_PLUGIN_VERSION: 1.12.1
-  GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GRADLE_ENTERPRISE_ACCESS_KEY }}
+  DEVELOCITY_INJECTION_ENABLED: true
+  DEVELOCITY_URL: https://ge.solutions-team.gradle.com
+  DEVELOCITY_PLUGIN_VERSION: 3.16.1
+  DEVELOCITY_CCUD_PLUGIN_VERSION: 1.12.1
+  DEVELOCITY_ACCESS_KEY: ${{ secrets.GRADLE_ENTERPRISE_ACCESS_KEY }}
```

Comment:
I'm pretty sure this needs to be `secrets.DEVELOCITY_ACCESS_KEY`, since that's the name of the secret as passed to this workflow.

Reply:
Right, fixed